### DISCUSSION TITLE
data.vulnerability.status "solved" and "active" at same time.

### DISCUSSION THREAD

**Discussion by lailtonmontenegro** [wazuh/wazuh#13955] [Q&A] [answered]
Hey guys, someone can help me understand whats it happened here? At the same moment the vulnerability alerts show me the vulnerability status as Solved and Active. Other events have the same situation. I'm not understanding. Please some one can help me?   

![image](https://user-images.githubusercontent.com/51930311/175117044-6458f6b3-cbdd-44fa-b198-13b797eac228.png)

**Comment by mdiego92** [comment, marked as answer]
Hello lalitomontenegro,

The Wazuh's Vulnerability Detector module may discover vulnerabilities in the applications installed on the agents. To discover vulnerabilities, agents can collect a list of installed apps (System inventory) and communicate it to the manager on a regular basis. The manager also creates a global vulnerability database from publicly available CVE repositories, which it then uses to cross-correlate with the agent's application inventory data. The information of vulnerabilities is the CVEs from Software installed where the agent is located.

If Wazuh-Dashboard shows that message it means that the vulnerability is solved. The column `data.vulnerability.status` will let you know if the vulnerability is active or is already solved within the installed application, **libssl1.1** in this case.

Please let us know if you have any remaining questions or doubts.

Thank you very much.

**Comment by lailtonmontenegro** [comment]
Nice @mdiego92, I got it! 

Tks for your help. :)
